boards/shields/amg88xx/doc/index.rst

```diff
@@ -1,71 +1,137 @@
+The Panasonic Grid-EYE is an infra-red sensor, which is comprised of 64 individual thermopile elements arranged in an 8x8 matrix.
+
+The sensor comes in a compact surface mount package and can be very easily integrated into thermometry applications.
+
+Grid-EYE is available in different varieties which differ in field of view, amplification factor and operating voltages. For example AMG8833 features a 60 degree field of view, a high gain amplification factor and 3.3V operating voltage.
+
+For the different part numbers that follow the AMG88xx scheme please refer to `IR Thermophile Array Sensor - Grid-EYE`_.
+
+For sensor evaluation and rapid prototyping multiple shields are available:
+
+- `Panasonic Grid-EYE Evaluation Shield`_
+
+ .. figure:: AMG88XX_panasonic_grid_eye_evaluation_shield.png
+     :width: 300px
+     :align: center
+     :alt: Panasonic Grid-EYE Evaluation Shield
 
-- `AMG88xx Evaluation Kit`_
 - `Adafruit AMG8833 8x8 Thermal Camera Sensor`_
 
-Current supported shields
-=========================
+ .. figure:: AMG8833_adafruit_ir_thermal_camera_sensor.png
+      :width: 200px
+      :align: center
+      :alt: Adafruit AMG8833 8x8 Thermal Camera Sensor
+
+Please note that the former follows the Arduino shield standard, while the latter is a breakout board only. Please use the ``amg88xx_grid_eye_eval_shield`` Grid-EYE shield designation for these boards.
+
+For the Panasonic Grid-EYE sensor an older evaluation kit was available, that also could be used as an Arduino shield.
+
+- Panasonic Grid-EYE Evaluation Kit
+
+ .. figure:: AMG88XX_panasonic_grid_eye_evaluation_kit.png
+      :width: 250px
+      :align: center
+      :alt: Panasonic Grid-EYE Evaluation Kit (deprecated)
+
+Please note that this evaluation kit is no longer available and no online documentation is available anymore. Please use the ``amg88xx_eval_kit`` Grid-EYE shield designation for this board instead.
+
+Requirements
+************
+
+These shields can only be used with a board that has Arduino connectors and where node aliases for I2C and GPIO interfaces are correctly defined in the board's device tree.
+
+This is true for example for Panasonic's PAN1780/PAN1770, PAN1781 and PAN1782 evaluation boards (`Panasonic Wireless Connectivity Development Hub`_).
 
-+--------------------------------+--------------------------------+
-| Name                           | Shield Designation             |
-|                                |                                |
-+================================+================================+
-| AMG88xx Evaluation Kit         | amg88xx_eval_kit               |
-+--------------------------------+--------------------------------+
+Jumper Configuration
+********************
 
-Requirements AMG88xx Evaluation Kit
-***********************************
+Panasonic Grid-EYE Evaluation Shield
+====================================
 
-On the AMG88xx Evaluation Kit, all jumpers except the J11 and J14 must be removed.
```

Comment:
This information seems to be lost.

Reply:
Fixed in [b040a75](https://github.com/zephyrproject-rtos/zephyr/pull/53244/commits/b040a75cf41d69878d2e601fda12d6fb097f6aef)